modules/logging-create-clf.adoc

```diff
@@ -19,36 +16,32 @@ You need administrator permissions for the namespace where you create the `Clust
 .ClusterLogForwarder resource example
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
-metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+metadata:  
+  name: <log_forwarder_name>
+  namespace: <log_forwarder_namespace>
 spec:
-  serviceAccountName: <service_account_name> <3>
+  managementState: Managed
+  outputs:
```

Comment:
I don't know that a filterSpec is necessary, but I think it would be helpful to add an 'inputs' section, rather than relying on the fact the reserved name is specified in the inputRefs, ex: 

```  
  inputs:
    - name: my-infra-input
      type: infrastructure
      infrastructure:
        sources:
          - node
          - container
```
We want to try and enforce "explicit" configurations

Reply:
added filterSpec and input